codegen/generator/nodejs/templates/src/object.ts.tmpl

```diff
@@ -0,0 +1,23 @@
+{{ define "object" }}
+	{{- with . }}
```

Comment:
> {{with pipeline}} T1 {{end}}
>>	If the value of the pipeline is empty, no output is generated;
>>	otherwise, dot is set to the value of the pipeline and T1 is
>>	executed.

Isn't `with .` a no op, setting `.` to `.`?

Reply:
It sets . to ., only if . is not `nil`.
If it's `nil`, it skips the whole `with` block.

Reply:
https://go.dev/play/p/2Agpw0q2uen